models/stg_intercom.yml

```diff
@@ -199,9 +235,6 @@ models:
         description: The id representing the parent conversation.
       - name: author_id
         description: The id representing the individual that sent the message.
-        tests:
```

Comment:
We _can_ add them as custom tests, but as I was scoping this out the actual impact of the test was pretty immaterial and felt it would be appropriate to simply remove the test. If this is something we want to add as a custom test I would say this is out of scope for these changes as they are needed ASAP.